app/viewModels/components/InputYesNoViewModel.scala

```diff
@@ -37,4 +37,9 @@ object InputYesNoViewModel {
   case class YesNoWithLegend(
     legend: String
   ) extends InputYesNoViewModel
+
+  case class YesNoWithLegendAndAdditionalHtml(
+    legend: String,
+    additionalHtml: Html
+  ) extends InputYesNoViewModel
```

Comment:
Can this be deleted?

Reply:
Yes, my bad! We didn't end up implementing this approach but seems to have slipped through the net